src/integrationTest/java/uk/gov/hmcts/reform/iacasepaymentsapi/testutils/WithIdamStub.java

```diff
@@ -0,0 +1,75 @@
+            + "MCwiYXVkaXRUcmFja2luZ0lkIjoiYjAxMjYxMzgtYWUwMC00ZmQ5LWFlNGEtZWQyNjUyODExMW"
+            + "E4LTI1MDE1MTY3IiwiaXNzIjoiaHR0cHM6Ly9mb3JnZXJvY2stYW0uc2VydmljZS5jb3JlLWNv"
+            + "bXB1dGUtaWRhbS1hYXQyLmludGVybmFsOjg0NDMvb3BlbmFtL29hdXRoMi9yZWFsbXMvcm9vdC"
+            + "9yZWFsbXMvaG1jdHMiLCJ0b2tlbk5hbWUiOiJhY2Nlc3NfdG9rZW4iLCJ0b2tlbl90eXBlIjoi"
+            + "QmVhcmVyIiwiYXV0aEdyYW50SWQiOiJwTExqdmNQWWl1TWFXd3hrbFpXUzJNV3FaZVEiLCJhdW"
+            + "QiOiJ4dWl3ZWJhcHAiLCJuYmYiOjE1OTA2NzQ0ODYsImdyYW50X3R5cGUiOiJhdXRob3JpemF0"
+            + "aW9uX2NvZGUiLCJzY29wZSI6WyJvcGVuaWQiLCJwcm9maWxlIiwicm9sZXMiLCJjcmVhdGUtdX"
+            + "NlciIsIm1hbmFnZS11c2VyIl0sImF1dGhfdGltZSI6MTU5MDY3NDQ4NiwicmVhbG0iOiIvaG1j"
+            + "dHMiLCJleHAiOjE1OTA3MDMyODYsImlhdCI6MTU5MDY3NDQ4NiwiZXhwaXJlc19pbiI6Mjg4MD"
+            + "AsImp0aSI6Ik05d2t1TEhMWnk3SEpLWWk5ZDBOOFdfcjEtMCJ9.ed7TiNviY0HPRNVu80M0Xe7"
+            + "2am_2HXeJ_2LUxWM4sXOAktqqyShFGmwC-NcMEgsRvASFT3DUaAgpuOZkypKtv2Lw3T7NJigfV"
+            + "kiod3y2G-3BeRq-0otx7gm4-D6UBmJ7vKuct5I9qZsAjYYCb8vMQQAU-sAnD6RfPeqA8kLxuDK"
+            + "vePrj_7ntEap-s-2jcLVLf4W2MU6VaZFK4fs99daRtn6eHQWU0PwirIxa0miuzu3AenyMi43K9"
+            + "NemUma5lnIn_UU-_AxytfYoFBuVw5eKtmjEPW6H1foxjFqbM8ovKEhD7Iod7Rn1R6mQMhZpesk"
+            + "pQkutWvaViejNfM-lhO-eaQ";
+
+    String EMAIL = "ia.legalrep.orgcreator@gmail.com";
+
+    default void addIdamTokenStub(WireMockServer server) {
+        server.addStubMapping(
+            new StubMapping(
+                newRequestPattern(RequestMethod.POST, urlEqualTo("/idam/o/token"))
+                    .withHeader("Content-Type", equalTo("application/x-www-form-urlencoded;charset=UTF-8"))
+                    .withRequestBody(
+                        equalTo("grant_type=password"
+                                + "&redirect_uri=http%3A%2F%2Flocalhost%3A3002%2Foauth2%2Fcallback"
+                                + "&client_id=ia"
+                                + "&client_secret=something"
+                                + "&username=ia-system-user%40fake.hmcts.net"
+                                + "&password=London05"
+                                + "&scope=openid+profile+roles"
+                        )
+                    )
+                    .build(),
+                aResponse()
+                    .withStatus(200)
+                    .withHeader("Content-Type", "application/json")
+                    .withBody("{\"access_token\": \"" + SYSTEM_USER_TOKEN + "\"}")
+                    .build()
+            )
+        );
+    }
+
+    default void addUserInfoStub(WireMockServer server) {
+        server.addStubMapping(
+            new StubMapping(
+                newRequestPattern(RequestMethod.GET, urlEqualTo("/idam/o/userinfo"))
+                    .withHeader("Content-Type", equalTo("application/json"))
+                    .withHeader("Authorization", equalTo("Bearer " + SYSTEM_USER_TOKEN))
```

Comment:
this line is not needed, we want to return same always same response for tests

Reply:
token reference line removed